app/main.py

```diff
@@ -1,2 +1,119 @@
+from typing import Any, Hashable
+
+
+class Node:
+    def __init__(self, key: Hashable, value: Any,
+                 next_node: "Node" = None) -> None:
```

Comment:
If argument can be None, you must show it in type hint
```suggestion
                 next_node: "Node" | None = None) -> None:
```

Reply:
Done.